app.js

```diff
@@ -0,0 +1,45 @@
+import { compareNumbers, getRandomNumber } from './Utils.js';
+
+const guessField = document.getElementById('guessField');
+
+const button = document.getElementById('button');
+
+const textResult = document.getElementById('text-result');
+
+const answer = document.getElementById('answer');
+
+const randomNumber = getRandomNumber();
```

Comment:
This is a great idea. Making a variable by the calling your function. Great job.